tests/test_SequenceManager.cc

```diff
@@ -832,3 +834,56 @@ TEST_CASE("SequenceManager: git repository", "[SequenceManager]")
     }
 
 }
+
+TEST_CASE("SequenceManager: fail creating sequence", "[SequenceManager]")
+{
+    const auto dir = temp_dir / "store_wp";
+    std::filesystem::create_directory(dir);
```

Comment:
We could remove a bit of syntax by doing a `using namespace std::filesystem` or `namespace fs = std::filesystem`.

Reply:
I must admit I like explicitly writing for example `std::size_t` instead of `size_t`.

Here I would use some abbreviation if it is used a lot. Which is (almost?) the case here.
On the other hand it's just these two occurrences.

Then I checked the rest of this repo and we use `filesystem` in other places, and everywhere with full namespace path.

`using fs = std::filesystem` has a certain Python feel :cringe:.

So for now I would keep as is.